packages/shared/components/atoms/buttons/SendButton.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="typescript">
     import { localize } from '@core/i18n'
+    import { resetNewTransactionDetails } from '@core/wallet'
     import { openPopup } from '@lib/popup'
     import { OnboardingButton } from 'shared/components/atoms'
 
-    function handleSendClick() {
+    function handleSendClick(): void {
```

Comment:
Can you rename this (and similarly any other click handlers) to `onSendClick`. Not in the conventions, but we use `onXYZ` for some component props, just helps to make it cleaner.